python/pyarrow/_parquet.pyx

```diff
@@ -1187,7 +1195,6 @@ cdef class ParquetReader(_Weakrefable):
                          .ReadSchemaField(field_index, &out))
         return pyarrow_wrap_chunked_array(out)
 
-
```

Comment:
Style nit, but can you keep two blank lines between classes and between individual functions?